src/methods.jl

```diff
@@ -134,10 +152,11 @@ flat_correct(frame::AbstractArray, flat_frame::AbstractArray; kwargs...) = flat_
 """
     trim(frame::AbstractArray, idxs)
 
-Trims the frame to remove the region specified by idxs.
+Trims the `frame` to remove the region specified by idxs.
 
 This function trims the array in a manner such that final array should be rectangular.
 The indices follow standard Julia convention, so `(:, 45:60)` trims all columns from 45 to 60 and `(1:20, :)` trims all the rows from 1 to 20.
+The function also supports FITS-style indices.
```

Comment:
This docstring doesn't explain what FITS-style indices are, and especially that they must be a string

Reply:
I have planned to add up a documentation section for Indexing Styles before release. There I will explain about all kinds of indexing styles that we support.